plugins/woocommerce/src/Admin/API/OnboardingProfile.php

```diff
@@ -309,6 +332,44 @@ public function get_profile_progress( $request ) {
 		return rest_ensure_response( $onboarding_progress );
 	}
 
+
+	/**
+	 * Update store's currency and measurement units.
+	 *
+	 * @param  WP_REST_Request $request Request data.
```

Comment:
Let's be more specific with the docs for the input, so that there's no confusion about what it's expecting.

Reply:
I've updated the [Add comment about required country_code param](https://github.com/woocommerce/woocommerce/pull/55463/commits/3b11a150feafbae15105b7bf72ef9ab3d01fac41) to note that `country_code` is required, but it is already documented in https://github.com/woocommerce/woocommerce/pull/55463/files/fca4956cd4cf0c5b2c230b7da385b4753986a880#diff-9912b4ac5d173e2c1433eef251f3e1289ed309d047de13107601b4e508b41db9R134